src/express.re

```diff
@@ -1,23 +1,27 @@
-let module Next = {
+module Next = {
   type t;
 };
 
-let module Request = {
+module Request = {
   type t;
 };
 
-let module Response = {
+module Response = {
   type t;
   external sendFile : t => string => 'a => unit = "" [@@bs.send];
-  external json: t => 'a => unit = "" [@@bs.send];
+  external sendString : t => string => unit = "send" [@@bs.send];
+  external sendObject : t => Js.t 'a => unit = "send" [@@bs.send];
```

Comment:
I think the technical term is "an object with row polymorphism lifted to Js.t", but yea pretty much :)

I think it's ok to track master for unstable bindings. And even bs releases are pretty frequent, so it likely wouldn't have to track master for very long. This change was merged into bs yesterday btw. Things move fast :)